src/main/java/book/store/dto/cartitem/CartItemDto.java

```diff
@@ -0,0 +1,9 @@
+package book.store.dto.cartitem;
+
+import lombok.Data;
+
+@Data
+public class CartItemDto {
```

Comment:
You should have CartItemRequestDto (bookId, quantity), UpdateCartItemDto(quantity) and response CartItemDto(id, bookId, bookTitle, quantity).